src/map/map.jl

```diff
@@ -0,0 +1,20 @@
+export get_field, set_field!
+
+################################################################################
+#
+#  All maps
+#
+################################################################################
+
+get_field(M::AbstractAlgebra.Map, f) = getfield(M, f) # fall back to Julia builtin
```

Comment:
These should definitely not be needed, as they are already defined in AbstractAlgebra. My guess is we need to import them from AbstractAlgebra (or Base) after all.

Reply:
Should work now once my pull request into AbstractAlgebra is merged.